app/assets/javascripts/reservations.js

```diff
@@ -8,6 +8,7 @@ $(document).ready(function() {
   function init_datepickers() {
     if (typeof minDaysFromNow == "undefined") {
       window['minDaysFromNow'] = 0;
+      window['maxDaysFromNow'] = 365;
```

Comment:
It's probably fine since we'll probably always have neither or both, but to be on the safe side, can we do the two variables in separate `typeof` `undefined` blocks.